src/models/unitConverters.ts

```diff
@@ -115,6 +115,15 @@ export class UnitConverter {
     this.inverseLinearScale(this.frequencyDomain.value, this.frequencyRange.value, this.frequencyInterpolator.value),
   );
 
+  public annotationRect(): Rect<Pixel> {
```

Comment:
is it worth reusing DOMRect?

Reply:
I have a couple of reasons for not using DOMRect

- We have have generic type hints (unfortunately not type checking) for a `Rect<Pixel>`
- The DOMRect is defined externally by the tc39 group and the TypeScript team, meaning that if we update TypeScript, DOMRect might change. This would sometimes be a benefit, but in this case, the `Rect` is not strongly dependent on the shape of the `DOMRect` standard
- DOMRect has some additional properties beyond x (left), y (top), width, and height (see image below)

![image](https://github.com/user-attachments/assets/e6614993-01f1-4d2b-98dc-0ce483d90c3f)